src/vma/util/sys_vars.h

```diff
@@ -587,9 +587,9 @@ extern mce_sys_var & safe_mce_sys();
 #define MCE_DEFAULT_CQ_AIM_INTERVAL_MSEC		(250)
 #define MCE_DEFAULT_CQ_AIM_INTERRUPTS_RATE_PER_SEC	(5000)
 #define MCE_DEFAULT_CQ_POLL_BATCH			(16)
-#define MCE_DEFAULT_PROGRESS_ENGINE_INTERVAL_MSEC	(10)
+#define MCE_DEFAULT_PROGRESS_ENGINE_INTERVAL_MSEC	(0)
```

Comment:
This change can cause TCP hang scenarios under loads in VMA.
If application is blocked on write, and waiting for ACK to arrive, so no other context will check the Rx CQ to handle this ingress ACK packet.
While it can help simple ping-pong latency test scenarios.